pnpbridge/docs/author_adapter.md

```diff
@@ -0,0 +1,486 @@
+# Extend the IoT Plug and Play bridge
+
+The [IoT Plug and Play bridge](https://docs.microsoft.com/azure/iot-pnp/concepts-iot-pnp-bridge#iot-plug-and-play-bridge-architecture) lets you connect the existing devices attached to a gateway to your IoT hub. You use the bridge to map IoT Plug and Play interfaces to the attached devices. An IoT Plug and Play interface defines the telemetry that a device sends, the properties it synchronizes with the cloud, and the commands that it responds to. You can install and configure the open-source bridge application on Windows or Linux gateways. Additionally, the bridge can be run as an Azure IoT Edge runtime module.
```

Comment:
The [IoT Plug and Play bridge](https://docs.microsoft.com/azure/iot-pnp/concepts-iot-pnp-bridge#iot-plug-and-play-bridge-architecture) or PnP Bridge lets you connect the existing devices attached to a gateway to your IoT hub. You use the PnP Bridge to map IoT Plug and Play interfaces to the attached devices. An IoT Plug and Play interface defines the telemetry that a device sends, the properties it synchronizes with the cloud, and the commands that it responds to. You can install and configure the open-source PnP Bridge application on Windows or Linux gateways. Additionally, the PnP Bridge can be run as an Azure IoT Edge runtime module.